README.md

```diff
@@ -16,6 +16,9 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
 - `odf subvolume delete <subvolume> <filesystem> <subvolumegroup>`: Deletes the stale subvolumes
 - `odf mons`
   - `restore-quorum <monID>`: Restore the mon quorum based on a single healthy mon since quorum was lost with the other mons
+- `odf operator`:
+  - `rook`:
```

Comment:
```suggestion
  - `rook`:
```

@travisn should I add restart sub-command to restart rook operator

Reply:
Yes, makes sense to me